pkg/termcorder/termcorder.go

```diff
@@ -0,0 +1,210 @@
+package termcorder
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+
+	"github.com/creack/pty"
+	"golang.org/x/term"
+)
+
+//Termcording represent a command/terminal recording
+type Termcording struct {
+	Config *Config
+	cmd    *exec.Cmd
+	out    io.Writer
```

Comment:
Shouldn't we have _two_ writers here, one for the terminal, one for the record file?

Reply:
Did you mean so we can (optionally) set the recording destination (file) in the constructor? If manually set,`out` here would replace the entire`io.MultiWriter`(set in line #161), redirecting all output to our `io.Writer` (e.g. when testing). Perhaps you think that instead of doing that, we should be able to set the `io.MultiWriter`'s  writers individually?